autowarefoundation/autoware
# Feature/update health checker threashold by rosparam

## Status
*DEVELOPMENT**

## Description
autowarefoundation/autoware_ai#605 

## Steps to Test or Reproduce
you should set /enable_statistisc rosparam before you launch all nodes

```
roslaunch autoware_health_checker health_checker.launch
```
update default.yaml in autoware_health_checker and 
```
rosparam load default.yaml
```
